src/scripts/main.js

```diff
@@ -1 +1,19 @@
 'use strict';
+
+const gallery = document.querySelector('.gallery');
+
+const changeImage = (eventElement) => {
+  const thumb = eventElement.target.closest('.gallery__thumb');
+
+  if(!thumb) {
+
+    eventElement.preventDefault();
+    const largePrevious = document.querySelector('.gallery__large-img');
+
+    largePrevious.src = eventElement.target.src;
+  } else {
+    return;
+  }
+};
```

Comment:
This function is meant to change the image displayed in the gallery when a thumbnail is clicked. However, the logic seems to be inverted. The code inside the `if` block should execute when `thumb` is truthy, not when it's falsy. You're currently preventing the default event and changing the image source when the clicked element is not a thumbnail, which is likely not the intended behavior.